src/GSoft.AspNetCore.Authentication.ClientCredentialsGrant/GSoft.AspNetCore.Authentication.ClientCredentialsGrant.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.16" />
```

Comment:
I wonder if we should enforce a 6.0.0 baseline as we do for Microsoft.Extensions.* libraries, or if we want to enforce a specific/latest version for these Microsoft.AspNetCore.* packages

Reply:
Umm true we keep Microsoft.Extension.* at 6.0.0. I thought we updated patches but that's just for sdk, for consistency we could keep it a 6.0.0

Reply:
Alright, after discussion we'll stay with a baseline at 6.0.0. A story will be created in order to update the renovate config so it considers this namespace like Microsoft.Extensions.*